resnet/resnet_main.py

```diff
@@ -163,13 +162,17 @@ def evaluate(hps):
         tag='Best Precision', simple_value=best_precision)
     summary_writer.add_summary(best_precision_summ, train_step)
     summary_writer.add_summary(summaries, train_step)
-    tf.logging.info('loss: %.3f, precision: %.3f, best precision: %.3f\n' %
-                    (loss, precision, best_precision))
+    output_str = 'loss: %.3f, precision: %.3f, best precision: %.3f\n' % \
+                 (loss, precision, best_precision)
+    tf.logging.info(output_str)
+    print(output_str)
     summary_writer.flush()
```

Comment:
Didn't work for me. This is what I ran:
`bazel-bin/resnet/resnet_main --eval_data_path=cifar-10-batches-bin/test_batch.bin --log_root=resnet_model --eval_dir=resnet_model/test --mode=eval --dataset='cifar10' --num_gpus=0 --logtostderr`